meetings/2024-03-06.md

```diff
@@ -45,12 +50,93 @@ URL: https://github.com/solid/specification/discussions/618
+* HZ: I'm understanding different themes. From TBL, There is no need for re-write but just minor changes. TBL?
+* TBL: Changes are addressing some of the questions that they (AC?) had. One of them was about dependencies. Some are for the REC track and so we need to prepare to move those things. Recognise them. The charter didnt' call that out. I pushed back on one point, the CSS WG for example, where it was raised to charter a general problem. But, it is not called the "Style System Where the User Can Something Output" WG. They didn't want other systems to be in there. So, like with VC. It is a spec. You could say that ensuring the certain things by people and cryptographically, ??? to do that. One of the htings is about the Mission and I'm happy to change that. Maybe not easy to do. From "Fix the Solid Spec" to "World of Read-Write Authenticated Data" would be fine.
+* PAC: TBL and I are saying the same thing I think we change the charter and for the Council to decide any way. Please adjust and then you can go. Truly this is not TBL's recommendation but ???. I still have some changes to push to GitHub and submit to the CG. I'm currently re-writing the Mission. So, TBL, this WIP.
+* TBL: Ok, thanks.
+* eP: I do support some natural language on read-write protected data. Nasty contrast with open data. So here is more protected resources. I participated in the FedCM WG proposal. There is possible overlap. There is no problem with it but Solid WG was pointe out. Especially with things like Solid-OIDC. If FedCM WG goes ahead then we might have alternative to Solid-OIDC. We should at least consider listing them as a possible Group to coordinate with.
+* HZ: Do you still want to do the poll for changing the name?
+* SC: Thinking about the name depends on the scope/mission. What do you want to get out of this poll 
+* PAC: Whether people think it is important to stick to "Solid" or not. I agree that the name depends on the mission. At this point do we feel we should stick with Solid or change.
+* HZ: I'm neutral but biased towards keeping the name because people are familiar with it. I don't know if you find a name that reflects.
+* SC: I don't have a strong opinion, I don't think it matters. In SocialWG, there was ActivityPub, AS2, LDN etc. Similarly this WG could publish a set of spec. In SocialWG some specs were overlaping. I don't think that the name has branding value to the working group. At some point it will be about sucess of the spec. I'm open to the idea to pick a different name. I think about Socially aware web, or talking about fixing the Web. I'm open to changing the name. Depends on the scope/mission/deliverables.
```

Comment:
```suggestion
* HZ: I'm neutral but biased towards keeping the name because people are familiar with it. I don't know if you can find another name that reflects the same.
* SC: I don't have a strong opinion, I don't think it matters. In SocialWG, there were ActivityPub, AS2, LDN, etc. Similarly this WG could publish a set of specs. In SocialWG, some specs were overlapping. I don't think that the name has branding value to the working group. At some point it will be about sucess of the spec. I'm open to the idea of picking a different name. I think about Socially aware web, or talking about fixing the Web. I'm open to changing the name. Depends on the scope/mission/deliverables.
```